### ISSUE TITLE
Overhaul rating system

### ISSUE THREAD

**Issue by RubenKelevra** [ajayyy/SponsorBlock#1283]
I noticed the tendency of users to rate cut marks negatively, but very very rarely positively. So two users unhappy with a cutmark let it disappear forevery while 100 happy users which don't notice anything negatively and thus do not rate it down doesn't change anything about that.

### New user protection:

I feel like we might "degrade" the users feedback by a multiplier below 100 own contributions. So we just do `if contributions < 100: rating = 1*(contributions/100)`.

So new users cannot be destructive by just voting stuff negatively without having some own contributions to the database.

### View vs votes
Additionally, we might want to assume a positive "ghost feedback" when 20 users watched a section, just for the "disappearing" rule. So if there are two negative votes but 40 views the net zero for the disappearing rule would be 0.

**Comment by ajayyy** [comment]
I disagree with thing, If anything downvotes aren't powerful enough

**Comment by RubenKelevra** [comment]
@ajayyy could you elaborate?

**Comment by ajayyy** [comment]
It is common for imprecise segments to be downvoted only once

**Comment by ajayyy** [comment]
I am not making sponsorblockcoins lol

**Comment by RubenKelevra** [comment]
Hmmmm... yeah just throwing ideas around 😂

Or maybe just have the popup allowing to vote not go away for a minute if the user is not voting enough? 🤔

**Comment by sup2069** [comment]
I like the idea. It reminds me of stackexchange's reputation levels to perform certain tasks.

**Comment by RubenKelevra** [comment]
> I like the idea. It reminds me of stackexchange's reputation levels to perform certain tasks.

Yeah, it's similar. The main issue is just that a user needs to install the add-on on two devices to randomly "delete" segments if he doesn't like the work (e.g. a sponsor removing the skips for their products or a YouTuber who doesn't like the add-on).

So a minimum amount of submissions to have your votes (at least the negative ones) have that large of an impact is most certainly necessary in the future to avoid abuse.

And 100 submissions isn't really a hard task, it will just naturally happen over time if you use the plugin as it's intended.

**Comment by RubenKelevra** [comment]
An example where IMHO the voting was just wrong:

I labeled a section as "Unpaid/Self promotion" in this video (and another used did the same), both times it got voted out of the database:

https://sb.ltn.fi/video/hE3KjKg69ZA/

Despite he mentioned in this section with whom he collaborated and give him a shoutout – which is not necessary to understand the main video.

The wiki list this as valid content for beeing marked as "Unpaid/Self promotion":
```
- Information about whom they collaborated with
```
